### DISCUSSION TITLE
problem with arrays of uuid's

### DISCUSSION THREAD

**Discussion by ghost** [supabase/supabase#2143] [Questions] [answered]
I am having a problem with adding uuids to an array, it always tells me it is an invalid array. is there a secret to this?

**Comment by steve-chavez** [comment, marked as answer]
Hey @pearlslugs,

How are you trying to add the uuids to the array? (code snippet would be helpful) Also, what's the type of the array?

**Reply by ghost** [reply]
Hello, thank you for the reply. When trying to add them, I was forgetting the quotes.

For anyone else who had the same problem, the correct way is:

[
"Uuid",
]
